velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -364,16 +383,23 @@ VectorPtr VectorFuzzer::fuzzComplex(
 }
 
 VectorPtr VectorFuzzer::fuzzDictionary(const VectorPtr& vector) {
+  return fuzzDictionary(vector, vector->size());
+}
+
+VectorPtr VectorFuzzer::fuzzDictionary(
+    const VectorPtr& vector,
+    vector_size_t size) {
   const size_t vectorSize = vector->size();
-  BufferPtr indices = AlignedBuffer::allocate<vector_size_t>(vectorSize, pool_);
+  VELOX_DCHECK(!(vectorSize == 0 && size > 0));
```

Comment:
What you want to check here? I am assuming size should not be smaller than vector size given the current usage?

Reply:
No `size` is the number of indices and irrelevant to the size of underlying vector.  The only corner case here is we cannot build a non-empty dictionary out of an empty underlying vector.